### PR TITLE
Adds database section

### PR DESCRIPTION
**Pull request by butschster**
- Fixes namespaces from Spiral\Database to Cycle\Database

### CODE REVIEW

File: database/declaration.md [hunk 1]
```diff
@@ -1,35 +1,44 @@
-# Database Schema Declaration
```
Comment (butschster) [inline database/declaration.md:1]: Revert changes

File: database/introspection.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Database - Schema Introspection
```
Comment (butschster) [inline database/introspection.md:1]: Check diffs between old and current

File: database/migrations.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+# Database - Migrations
+Spiral ships with a set of embedded commands to control your database migrations, [component](https://github.com/cycle/migrations) 
+is build upon DBAL and supports virtual databases and prefixes.
+
+```php
+composer require cycle/migrations
```
Comment (butschster) [inline database/migrations.md:6]: spiral'migrations

File: database/migrations.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+```
+
+Migration state table can automatically initiate using the command `migrate:init`.
+
+## Generate Migrations
+You can automatically generate a set of migration files during schema compilation. In this case, you have the freedom 
+to alter such migrations manually before running them. To achieve that you must install the Cycle Migrations extension:
+
+```php
+composer require cycle/schema-migrations-generator
```
Comment (butschster) [inline database/migrations.md:34]: cycle/migrations

File: database/migrations.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+}
+```
+
+## Working with migrations
+You can run all outstanding migrations using `migrate` command.
+
+```php
+use Cycle\Migrations\Capsule;
+
+$migrator->run( new Capsule($dbal->database()));
```
Comment (butschster) [inline database/migrations.md:85]: Extra space

File: database/migrations.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+```
+
+Use `-vv` flag to get more information about generated queries:
```
Comment (butschster) [inline database/migrations.md:88]: remove

File: database/migrations.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+[MySQLDriver] SELECT
+`id`, `time_executed`
+FROM `migrations`
+WHERE `migration` = 'my_migration'
+Migration my_migration was successfully executed.
+```
+
+> Enable DEBUG mode first.
+
+Run command `migrate:rollback` and `migrate:replay` to rollback your migrations.
```
Comment (butschster) [inline database/migrations.md:143]: remove

File: database/migrations.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+    {
+        $this->table('sample_table')
+            ->dropColumn('field')
+            ->update();
+    }
+}
+```
+
+```
+$ php app.php migrate -vv
```
Comment (butschster) [inline database/migrations.md:168]: remove

File: database/migrations.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+```php
+use Cycle\Migrations\Capsule;
+
+$migrator->run( new Capsule($dbal->database()));
+```
+
+Use `-vv` flag to get more information about generated queries:
+
+```bash
+$ php app.php migrate:replay -vv
```
Comment (butschster) [inline database/migrations.md:91]: remove

File: database/transactions.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
+# Database - Transactions
+Both Database and Driver instances provide access to transaction management for your queries.
+You can manage transactions manually or use pre-created Closure based flow.
+
+## Examples
+To start and commit transaction manually use `begin` and `commit` methods of Database:
+```php
+$db->begin();
```
Comment (butschster) [inline database/transactions.md:8]: Check method name

**Review by butschster — COMMENTED** [review]
Fill manifest file with new docs structure

File: advanced/sync-schema.md [hunk 11]
```diff
@@ -29,17 +29,17 @@ Such an approach is useful for development environments, but might cause issues
 You can automatically generate a set of migration files during schema compilation. In this case, you have the freedom to alter such migrations manually before running them. To achieve that you must install the Cycle Migrations extension:
 
 ```php
-composer require cycle/migrations
+composer require spiral/migrations
```
Comment (roxblnfk) [inline advanced/sync-schema.md:32]:
```suggestion
composer require cycle/migrations
```

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**